### PR TITLE
fix(expo): fix expo build-list executor and update examples

### PR DESCRIPTION
**Pull request by xiongemi**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/generated/packages/expo/documents/overview.md [hunk 1]
```diff
@@ -69,73 +69,133 @@ Replace `your-lib-name` with the app's name as defined in your `tsconfig.base.js
...
 
+{% /tab %}
+{% /tabs %}
+
 ### Install Compatible NPM Packages
 
 To install packages that is compatible with current version of Expo, run:
 
-```shell
+```bash
```
Comment (bcabanes) [inline docs/generated/packages/expo/documents/overview.md:183]: `shell` was the right choice though.

Comment (xiongemi) [reply docs/generated/packages/expo/documents/overview.md]: done. changed it back.

File: packages/expo/docs/build-list-examples.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "build-list": {
+      "executor": "@nrwl/expo:build-list",
+      "options": {}
+    }
+    //...
+  }
+}
+```
+
+```bash
```
Comment (bcabanes) [inline packages/expo/docs/build-list-examples.md:20]:
```suggestion
```shell
```

Comment (xiongemi) [reply packages/expo/docs/build-list-examples.md]: done. changed.

**Review by bcabanes — APPROVED** [review]

**Review by ndcunningham — APPROVED** [review]

**Pull request merged**